Detect new brands: Altibox, G Guard, Supermax, TV+, UnoPhone, WildRed and Detect devices for exist brands

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat(device) detect new brand TV+: Smart TV
feat(device) detect brand Panasonic: Smart TV
feat(device) detect brand Reeder: Smart TV
feat(device) detect brand Saba: 32SB6000H, 42SB6000F
feat(device) detect brand Erisson: 32LES905T2SM

ref #7882

sanchezzzhak committed:
feat(device) detect new brand Altibox: Smart TV
feat(device) detect new brand G Guard: Smart TV
feat(device) detect new brand Supermax: Smart TV
feat(device) detect brand Philips: NeoViu D1
feat(device) detect brand Dyon: 2K Smart TV
feat(device) detect brand Walton: 2K Smart TV
feat(device) detect brand Bell: Streamer
feat(device) detect brand XGIMI: MoGo Pro
feat(device) detect brand Telia: TV Box Nordic
feat(device) detect brand Polaroid: SmartBox S (SMX4K)

ref #7882

sanchezzzhak committed:
feat(device) detect new brand UnoPhone: Premier 5 Max, Premier 5
feat(device) detect new brand WildRed: 42QWS9001
feat(device) detect brand Xiaomi: Redmi Y3 (M1810F6G), 15 (24129PN74C), Redmi K80 Pro (24117RK2CC), 15 Pro
feat(device) detect brand Vivo: iQOO 13 (V2408A), iQOO Z9s Pro 5G (I2305)
feat(device) detect brand Realme: GT 7 Pro (RMX5011, RMX5010)
feat(device) detect brand Ghia: GK133M3
feat(device) detect brand Sky: Elite T10 Pro
feat(device) detect brand Ulefone: Tab A11 Pro
feat(device) detect brand Oukitel: OT11
feat(device) detect brand ONYX BOOX: Note Air 4C
feat(device) detect brand TOSCIDO: P101
feat(device) detect brand Huawei: Honor Magic 7 (PTP-AN00), Honor Magic 7 Pro (PTP-AN10), Nova 3i (IVY-LX2), Honor X9c (BRP-NX1), Honor X60 (BRC-AN00)
feat(device) detect brand Nubia: Z70 Ultra (NX733J)
feat(device) detect brand Google: Pixel 8 (GPJ41)
feat(device) detect brand VGO TEL: Flex 2
feat(device) detect brand Doogee: V Max Pro
feat(device) detect brand Unonu: N55S
feat(device) detect brand Meizu: Lucky 08
feat(device) detect brand Sparx: Neo 8 Lite
feat(device) detect brand POCO: C75
feat(device) detect brand JVC: Smart TV
feat(device) detect brand Rombica: 65MT-UDG54G
feat(device) detect brand ONVO: 24OV6000H
feat(device) detect brand Akai: TA43BU500, TA32BF550
feat(device) detect brand Erisson: 42FLES901T2SM
feat(device) detect brand Motorola: Moto G75 5G, Moto G45 5G, Moto G35 5G

sanchezzzhak committed:
feat(device) detect brand Asano: 32LH5000T 32.0"

liviuconcioiu commented on Tests/fixtures/tv-4.yml:
```diff
@@ -6186,3 +6186,417 @@
+    engine_version: 104.0.5112.97
+  device:
+    type: tv
+    brand: G Guard
```
```suggestion
    brand: G-Guard
```

sanchezzzhak committed:
Update Parser/Device/AbstractDeviceParser.php

Co-authored-by: Liviu-Mihail Concioiu <liviu.concioiu@gmail.com>

sanchezzzhak committed:
Update Tests/fixtures/tv-4.yml

Co-authored-by: Liviu-Mihail Concioiu <liviu.concioiu@gmail.com>

sanchezzzhak committed:
feat(device) detect brand BBK: 40LEX-9201/FTS2C, 40LEX-7291/FTS2C, 40LEX-7246/FTS2C, 32LEX-7292/TS2C, 32LEX-7291/TS2C, 32LEX-7288/TS2C, 32LEX-7264/TS2C, 32LEX-7264/TS2C, 32LEX-7260/TS2C,
32LEX-7259/TS2C, 32LEX-7257/TS2C, 32LEX-7249/TS2C, 32LEX-7246/TS2C, 32LEX-7213/TS2C, 32LEX-7212/TS2C, 32LEX-7204/TS2C, 32LEX-7202/TS2C, 24LEX-7289/TS2C
feat(device) detect brand Erisson: 40FLX9000CT2, 40FLES901T2SM, 32LES906T2SM
feat(device) detect brand BAFF: 32Y HD-R
feat(device) detect brand DEXP: 24HEY1

liviuconcioiu committed:
Merge branch 'master' into pr_device_477
